examples/sass-and-ts/README.md

```diff
@@ -4,16 +4,34 @@
 
 ![Sass test](images/sass.png)
 
-Note: run `npm install` in this folder to symlink the `cypress-react-unit-test` dependency.
+> Usage
```

Comment:
```suggestion
## Usage
```

By using headers it will create anchor links that other people can share as direct urls